README.md

```diff
@@ -1,33 +1,74 @@
-# payoutManagementSystem
+# PayoutManagementSystem
+
 
 This project is about the payout management system built using golang.
 
 # Project Setup
 
 ## Clone the repository
 
-Clone this repo: <a href = "https://github.com/Swarathmica-infraspec/payoutManagementSystem"> source link  </a>
+Clone this repo: <a href = "https://github.com/Swarathmica-infraspec/payout-management-system"> source link  </a>
 
 # Requirements
 
 GO-VERSION: 1.25.0
 
 The project contains payoutmanagementsystem/ <br>
 - .github/workflows/payoutManagementSystem.yml <br>
+- payee/
+  - payee.go <br>
+  - payee_test.go <br>
+  - payee_db.sql <br>
 - go.mod <br>
 - go.sum <br>
-- main.go <br>
-- main_test.go <br>
 - README.md <br>
 
 
-# Run Tests
+NOTE: Only email ids with .com are supported.
+
+
+
+# Database Setup
+
+We use PostgreSQL running inside Docker for persistant storage.
+
+## 1. Start Postgres with Docker Compose
+
+From the project root, run:
+
+docker compose up -d db
```

Comment:
[need-documentation-fix] tried following the instruction, but the command failed.
<img width="809" height="69" alt="Screenshot 2025-09-10 at 6 16 24 PM" src="https://github.com/user-attachments/assets/69981895-a62b-4109-9aa0-509d9e91677b" />